<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>822</width>
    <height>721</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>房贷计算器</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <widget class="QWidget" name="widget" native="true">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>610</y>
         <width>261</width>
         <height>61</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btn_Result">
          <property name="text">
           <string>开始计算</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btn_CleanAgain">
          <property name="text">
           <string>清空重算</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="widget_2" native="true">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>0</y>
         <width>541</width>
         <height>71</height>
        </rect>
       </property>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>30</y>
          <width>81</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>贷款类别</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_3">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>30</y>
          <width>111</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>商业贷款</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_4">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>30</y>
          <width>121</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>公积金贷款</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_5">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>30</y>
          <width>131</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>组合贷款</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="widget_6" native="true">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>70</y>
         <width>521</width>
         <height>281</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QStackedWidget" name="stackedWidget_4">
          <property name="currentIndex">
           <number>1</number>
          </property>
          <widget class="QWidget" name="page_8">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QWidget" name="widget_3" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <spacer name="verticalSpacer_17">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>计算方式</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>10</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QRadioButton" name="radioButton_6">
                 <property name="text">
                  <string>根据面积、单价计算</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QRadioButton" name="radioButton_7">
                 <property name="text">
                  <string>根据贷款总额计算</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer_18">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_9">
           <widget class="QWidget" name="widget_4" native="true">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>10</y>
              <width>161</width>
              <height>201</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <spacer name="verticalSpacer_3">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>商业贷款额</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_4">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>商业利率</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_5">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="label_14">
               <property name="text">
                <string>公积金总额</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_6">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="label_15">
               <property name="text">
                <string>公积金利率</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_7">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
           <zorder>widget_4</zorder>
           <zorder>stackedWidget_2</zorder>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QStackedWidget" name="stackedWidget_2">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="page_3">
           <widget class="QWidget" name="widget_8" native="true">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>40</y>
              <width>241</width>
              <height>211</height>
             </rect>
            </property>
            <widget class="QLabel" name="label_3">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>30</y>
               <width>71</width>
               <height>31</height>
              </rect>
             </property>
             <property name="text">
              <string>房屋单价</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_4">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>80</y>
               <width>71</width>
               <height>31</height>
              </rect>
             </property>
             <property name="text">
              <string>房屋面积</string>
             </property>
            </widget>
            <widget class="QLineEdit" name="lineEdit_2">
             <property name="geometry">
              <rect>
               <x>90</x>
               <y>90</y>
               <width>111</width>
               <height>22</height>
              </rect>
             </property>
            </widget>
            <widget class="QLabel" name="label_5">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>130</y>
               <width>71</width>
               <height>31</height>
              </rect>
             </property>
             <property name="text">
              <string>按揭成数</string>
             </property>
            </widget>
            <widget class="QComboBox" name="RestrictedMortgages">
             <property name="geometry">
              <rect>
               <x>90</x>
               <y>140</y>
               <width>111</width>
               <height>22</height>
              </rect>
             </property>
            </widget>
            <widget class="QLineEdit" name="lineEdit">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="geometry">
              <rect>
               <x>90</x>
               <y>40</y>
               <width>111</width>
               <height>21</height>
              </rect>
             </property>
            </widget>
            <widget class="QLabel" name="label_18">
             <property name="geometry">
              <rect>
               <x>210</x>
               <y>40</y>
               <width>41</width>
               <height>20</height>
              </rect>
             </property>
             <property name="text">
              <string>元</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_19">
             <property name="geometry">
              <rect>
               <x>210</x>
               <y>89</y>
               <width>41</width>
               <height>31</height>
              </rect>
             </property>
             <property name="text">
              <string>㎡</string>
             </property>
            </widget>
           </widget>
          </widget>
          <widget class="QWidget" name="page_7">
           <widget class="QWidget" name="widget_7" native="true">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>10</y>
              <width>201</width>
              <height>211</height>
             </rect>
            </property>
            <widget class="QLineEdit" name="lineEdit_8">
             <property name="geometry">
              <rect>
               <x>6</x>
               <y>26</y>
               <width>171</width>
               <height>22</height>
              </rect>
             </property>
            </widget>
            <widget class="QLineEdit" name="lineEdit_5">
             <property name="geometry">
              <rect>
               <x>6</x>
               <y>72</y>
               <width>171</width>
               <height>22</height>
              </rect>
             </property>
            </widget>
            <widget class="QLineEdit" name="lineEdit_6">
             <property name="geometry">
              <rect>
               <x>6</x>
               <y>117</y>
               <width>171</width>
               <height>22</height>
              </rect>
             </property>
            </widget>
            <widget class="QLineEdit" name="lineEdit_7">
             <property name="geometry">
              <rect>
               <x>6</x>
               <y>163</y>
               <width>171</width>
               <height>22</height>
              </rect>
             </property>
            </widget>
            <widget class="QLabel" name="label_11">
             <property name="geometry">
              <rect>
               <x>180</x>
               <y>30</y>
               <width>41</width>
               <height>20</height>
              </rect>
             </property>
             <property name="text">
              <string>元</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_12">
             <property name="geometry">
              <rect>
               <x>180</x>
               <y>120</y>
               <width>41</width>
               <height>21</height>
              </rect>
             </property>
             <property name="text">
              <string>元</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_16">
             <property name="geometry">
              <rect>
               <x>180</x>
               <y>80</y>
               <width>41</width>
               <height>20</height>
              </rect>
             </property>
             <property name="text">
              <string>%</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_17">
             <property name="geometry">
              <rect>
               <x>180</x>
               <y>170</y>
               <width>41</width>
               <height>20</height>
              </rect>
             </property>
             <property name="text">
              <string>%</string>
             </property>
            </widget>
           </widget>
          </widget>
          <widget class="QWidget" name="page_4">
           <widget class="QLabel" name="label_9">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>60</y>
              <width>71</width>
              <height>41</height>
             </rect>
            </property>
            <property name="text">
             <string>贷款总额</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="lineEdit_4">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>110</y>
              <width>161</width>
              <height>21</height>
             </rect>
            </property>
           </widget>
           <widget class="QLabel" name="label_20">
            <property name="geometry">
             <rect>
              <x>200</x>
              <y>109</y>
              <width>61</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string>万元</string>
            </property>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="widget_5" native="true">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>440</y>
         <width>251</width>
         <height>71</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <spacer name="horizontalSpacer_16">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>按揭年数</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_17">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QComboBox" name="MortgageYears"/>
        </item>
        <item>
         <spacer name="horizontalSpacer_18">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="widget_9" native="true">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>360</y>
         <width>331</width>
         <height>61</height>
        </rect>
       </property>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>71</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>贷款利率</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_3">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>20</y>
          <width>151</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_21">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>20</y>
          <width>51</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>%</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="widget_10" native="true">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>530</y>
         <width>321</width>
         <height>61</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <spacer name="horizontalSpacer_9">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>还款方式</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_10">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton">
          <property name="text">
           <string>等额本息</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_11">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_2">
          <property name="text">
           <string>等额本金</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_12">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="label_22">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>450</y>
         <width>71</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>年</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2">
      <widget class="QListWidget" name="listWidget">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>60</y>
         <width>421</width>
         <height>371</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_Calaulator">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>510</y>
         <width>131</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>返回上一页</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_Close">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>510</y>
         <width>131</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>关闭页面</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
